example_dags/example_sqlite_load_transform.py

```diff
@@ -12,12 +12,12 @@
 
 
 @aql.transform()
-def get_top_five_animations(input_table: Table):
+def get_top_five_animations(input_table: Table):  # skipcq: PYL-W0613
```

Comment:
`# skipcq: PYL-W0613` can be removed. I feel @kaxil  has added it globally.

Reply:
@sunank200 I only added this because DeepSource complained when I created the PR 3h ago. Did Kaxil do anything during this period?

Reply:
No didn't add anything in the meanwhile -- but we can do that tomorrow